### PR TITLE
Add missing symlink to generate "certs" state properly for SLE Micro 5.0 and openSUSE MicroOS (bsc#1188503)

### PR DESCRIPTION
**Pull request by meaksh**
## What does this PR change?

Currently the "certs" state is empty for SLE Micro 5.0 and openSUSE MicroOS instances, this means the SSL certificate from the Uyuni server is not automatically trusted on the SLE Micro instance once it's registered:

```console
# salt-call state.apply certs
local:

Summary for local
-----------
Succeeded: 0
Failed:   0
-----------
Total states run:    0
Total run time:  0.000 ms
```

This PR adds the missing symlink so the "certs" state is properly generated:

```console
# salt-call state.apply certs
local:
----------
          ID: /etc/pki/trust/anchors/RHN-ORG-TRUSTED-SSL-CERT
    Function: file.managed
      Result: True
     Comment: File /etc/pki/trust/anchors/RHN-ORG-TRUSTED-SSL-CERT is in the correct state
     Started: 16:49:17.677959
    Duration: 92.773 ms
     Changes:   
----------
          ID: update-ca-certificates
    Function: cmd.run
        Name: /usr/sbin/update-ca-certificates
      Result: True
     Comment: State was not run because none of the onchanges reqs changed
     Started: 16:49:17.774798
    Duration: 0.006 ms
     Changes:   

Summary for local
------------
Succeeded: 2
Failed:    0
------------
Total states run:     2
Total run time:  92.779 ms
```

## GUI diff

No difference.

- [x] **DONE**

## Documentation
- No documentation needed: only internal and user invisible changes

- [x] **DONE**

## Test coverage
- No tests: **we're not automatically testing SLE Micro - manual testing by QA**

- [x] **DONE**

## Links

Tracks https://github.com/SUSE/spacewalk/issues/15507

- [x] **DONE**

## Changelogs

If you don't need a changelog check, please mark this checkbox:

- [ ] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_pgsql_tests"
- [ ] Re-run test "schema_migration_test_oracle"
- [ ] Re-run test "schema_migration_test_pgsql"
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"
- [ ] Re-run test "spacecmd_unittests"

### CODE REVIEW

**Review by juliogonzalez — COMMENTED** [review]
I guess you use SLES12.sls because it's what we do for SLE15.sls (a symlink to SLES12.sls), right?

File: susemanager-utils/susemanager-sls/susemanager-sls.changes [hunk 1]
```diff
@@ -1,3 +1,4 @@
+- Add missing symlink to generate "certs" for SLE Micro 5.0 (bsc#1188503)
```
Comment (juliogonzalez) [inline susemanager-utils/susemanager-sls/susemanager-sls.changes:1]: It's not to generate certs, but to install them, right?

Comment (meaksh) [reply susemanager-utils/susemanager-sls/susemanager-sls.changes:1]: to generate the "certs" state for SLE Micro... I'm fixing this

**Comment by meaksh** [comment]
> I guess you use SLES12.sls because it's what we do for SLE15.sls (a symlink to SLES12.sls), right?

that's correct

**Review by juliogonzalez — APPROVED** [review]

**Pull request merged**